jwst/scripts/adjust_wcs.py

```diff
@@ -0,0 +1,320 @@
+
+    # Parse input parameters
+    parser = argparse.ArgumentParser(
+        prog="adjust_wcs",
+        description="Apply adjustments to data model's WCS"
+    )
+
+    parser.add_argument(
+        "arg0",
+        nargs='*',
+        metavar="input",
+        type=str,
+        help="Input data model file(s)."
+    )
+
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument(
+        "-u",
+        "--update",
+        action="store_true",
+        help="Update input data models."
+    )
+    group.add_argument(
+        "--suffix",
+        type=str,
+        help="Suffix for output files."
+    )
+    group.add_argument(
+        "-f",
+        "--file",
+        type=str,
+        help="Output file name."
+    )
+
+    parser.add_argument(
+        "--overwrite",
+        action="store_true",
+        help="Overwrite existing output files (ignored with -u/--update)."
+    )
+
+    parser.add_argument(
+        "-x",
+        "--expand",
+        action="store_true",
+        help="Expand wild cards in input file names (cannot be combined with "
+             "-f/--file)."
+    )
+
+    parser.add_argument(
+        "-w",
+        "--wcs",
+        action="store_true",
+        help="Save output WCS only to an ASDF file (ignored with -u/--update)."
+    )
+
+    group_wcs = parser.add_argument_group(
+        title='WCS adjustment parameters'
+    )
+    group_wcs.add_argument(
+        "-r",
+        "--ra_delta",
+        type=float,
+        default=0.0,
+        help="Delta RA (longitude), degrees."
+    )
+    group_wcs.add_argument(
+        "-d",
+        "--dec_delta",
+        type=float,
+        default=0.0,
+        help="Delta DEC (latitude), degrees."
+    )
+    group_wcs.add_argument(
+        "-o",
+        "--roll_delta",
+        type=float,
+        default=0.0,
+        help="Delta roll angle, degrees."
+    )
+    group_wcs.add_argument(
+        "-s",
+        "--scale_factor",
+        type=float,
+        default=1.0,
+        help="Scale factor."
+    )
+
+    parser.add_argument(
+        "-v",
+        "--version",
+        action="version",
+        version="v{0}".format(jwst.__version__)
+    )
+
+    options = parser.parse_args()
+
+    if options.file:
+        if options.expand:
+            parser.error(
+                "argument -x/--expand: not allowed with argument -f/--file"
+            )
+        if len(options.arg0) > 1:
+            parser.error(
+                "only one input file allowed with argument -f/--file"
+            )
+
+    wcs_pars = [
+        options.ra_delta,
+        options.dec_delta,
+        options.roll_delta,
+        options.scale_factor
+    ]
+
+    if wcs_pars == [0.0, 0.0, 0.0, 1.0]:
+        logger.warning(
+            "All WCS adjustment parameters (delta RA, DEC, roll, and scale "
+            "factor) have default values (identical transformation)."
+        )
+        logger.warning("Nothing to do. Quitting.")
```

Comment:
I think we are using the `warnings` module for warnings elsewhere, so let's stick to that.

Reply:
`logger.warning` or `log.warning` are throughout the pipeline code. Just an example: